### DISCUSSION TITLE
Line blame does not stay on after toggled

### DISCUSSION THREAD

**Discussion by lsliwaradioluz** [gitkraken/vscode-gitlens#1678] [Q&A] [answered]
Hello team,

Today I've noticed that line blame stopped appearing in files I am editing using vscode. Earlier GitLens was working on files automatically every time I opened one, now I need to execute the `GitLens: Toggle line blame` command from the palette to actually turn it back on. Unfortunately, it is off again after I reload the window. I would like it to be always on - just the way it used to be.

Moreover, I've also noticed that hovers do not work anymore. I can't see the window containing the detailed blame information. Here's the pain I experience in its entirety:

https://www.loom.com/share/62f0cac41c914ad38cbd4312ff2c3d8e

My GitLens extension version is `11.6.1` and my vs code is the latest `September 2021 (version 1.61)`. I've tried to downgrade both two versions down but - unfortunately - this didn't work. Perhaps it's just some settings of mine getting messed up but I can't locate the bug.

Is there any other information I could provide that would come in handy for solving the issue?

**Comment by lsliwaradioluz** [comment, marked as answer]
Literally a minute after I'd created the ticket, I've become really suspicious about this setting:
![image](https://user-images.githubusercontent.com/39009379/136622415-7e40c0be-7e0d-494f-99b0-d9055fa63bb4.png)

It turns out it ain't `zen` by default. Resetting it resolved all of my aforementioned problems:
![image](https://user-images.githubusercontent.com/39009379/136622472-4099ce73-7e4d-4e9e-bda5-ac3bf9856e83.png)

Long story short: if you want to have your beautiful GitLens extension always displaying the blame (along with the details), you should never set the mode to `zen`. 😄
